cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -1530,39 +1538,45 @@ TYPED_TEST(ReductionTest, UniqueCount)
   std::vector<bool> host_bools({true, true, true, false, true, true, true, true});
   std::vector<T> v = convert_values<T>(int_values);
 
+  auto const output_type = cudf::data_type{cudf::type_to_id<cudf::size_type>()};
```

Comment:
Should we add a test with different output_types? Also a test for unsupported types?

Reply:
I can add a test for a different output-type. Unsupported type test is in the Errors test above.